### PR TITLE
[syncd][tests] Add syncd deprecated attribute value test

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: tests/BCM56850.pl [hunk 1]
```diff
@@ -594,7 +594,31 @@ sub test_bulk_object
...
+
+    my $key = `redis-cli -n 1 "keys" "*"|grep SAI_OBJECT_TYPE_NEXT_HOP_GROUP`;
+
+    chomp $key;
+
+    `redis-cli -n 1 hset "$key" "SAI_NEXT_HOP_GROUP_ATTR_TYPE" "SAI_NEXT_HOP_GROUP_TYPE_ECMP"`;
+
+    start_syncd_warm;
+
+    play "non_depreacated.rec", 0;
```
Comment (shi-su) [inline tests/BCM56850.pl:615]: Just wanted to confirm if 0 here means expecting 0 operations?

File: tests/BCM56850.pl [hunk 2]
```diff
@@ -594,7 +594,31 @@ sub test_bulk_object
...
+
+    exit;
```
Comment (shi-su) [inline tests/BCM56850.pl:617]: I suspect this will exit the whole script and skip everything afterward.

Comment (kcudnik) [reply tests/BCM56850.pl:617]: leftover from debugging :P

File: tests/BCM56850.pl (hunk 1 above)
Comment (kcudnik) [reply tests/BCM56850.pl:615]: yes, this is not the best design, i will have to redesign those tests some day

**Review by shi-su — APPROVED** [review]

**Pull request merged**